Library/Formula/gcc.rb

```diff
@@ -32,6 +32,15 @@ def osmajor
     sha256 "77bce635f78bc26bd01010b5ece480251af223bf2dba6d48c29af6b29b441296" => :mountain_lion
   end
 
+  if MacOS::Xcode.version > "7.0" || MacOS::CLT.version > "7.0"
```

Comment:
I'll just apply to `el_capitan`; I'm funny about applying patches when they aren't needed.